cephadm.yml

```diff
@@ -0,0 +1,149 @@
+---
+
+- name: preconfigure linodes
+  import_playbook: pre-config.yml
+
+- hosts: all
+  become: yes
+  tasks:
+  - name: fetch cephadm
+    delegate_to: localhost
+    register: cephadm
+    run_once: true
+    uri:
+      force: yes
+      mode: 0755
+      return_content: yes
+      url: https://github.com/ceph/ceph/raw/octopus/src/cephadm/cephadm
+
+  - name: distribute cephadm
+    copy:
+      dest: /root/cephadm
+      group: root
+      mode: 0755
+      owner: root
+      content: "{{ cephadm.content }}"
+
+  - name: create /etc/ceph directory
+    file:
+      path: /etc/ceph
+      state: directory
+      owner: root
+      group: root
+      mode: 0755
+
+
+- hosts: mon-000
+  become: yes
+  tasks:
+  - name: check ceph installed
+    stat:
+      path: /etc/ceph/ceph.conf
+    register: ceph_installed
+
+  - name: bootstrap octopus
+    shell: /root/cephadm bootstrap --allow-fqdn-hostname --mon-ip {{ monitor_address }}
+    when: not ceph_installed.stat.exists
+
+  - name: config public network
+    shell: /root/cephadm shell -- ceph config set global public_network 192.168.0.0/16
+
+  - name: config cluster_network
+    shell: /root/cephadm shell -- ceph config set global cluster_network 192.168.0.0/16
+
+  - name: log to files
+    shell: /root/cephadm shell -- ceph config set global log_to_file true
+
+  - name: log cluster to file
+    shell: /root/cephadm shell -- ceph config set global mon_cluster_log_to_file true
+
+  - name: disable stderr logging
+    shell: /root/cephadm shell -- ceph config set global log_to_stderr false
```

Comment:
interesting. Fyi, this is going to break `cephadm logs ...`.

Reply:
That's fine I think? I have a set of scripts which collect logs from the usual `/var/log/ceph`.